core/capabilities/syncer_test.go

```diff
@@ -1,18 +1,24 @@
-package capabilities_test
+package capabilities
```

Comment:
Why this change?

Reply:
This was so the test could access internal functions like sync()... this results in a cleaner test since the function higher up runs in a loop and logs errors; by dropping down to sync() we get errors back and the test is more deterministic too.